test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smtrcomp.py

```diff
@@ -405,17 +410,19 @@ def test_serving(self, sagemaker_session, ecr_image, framework_version, instance
                                framework_version=framework_version,
                                hyperparameters={
                                     TrainingCompilerConfig.HP_ENABLE_COMPILER : True,
+                                    'save-as-tf': True,
                                },
                                )
         estimator.fit(mnist_dataset, job_name=unique_name_from_base('test-TF-trcomp-serving'))
         _assert_model_exported_to_s3(estimator)
         captured = capsys.readouterr()
         _assert_training_compiler_invoked(captured)
-        predictor = estimator.deploy(initial_instance_count=1, instance_type=instance_type)
+        predictor = estimator.deploy(initial_instance_count=1, instance_type=instance_type,
+            image_uri=f'763104351884.dkr.ecr.{estimator.sagemaker_session.boto_region_name}.amazonaws.com/tensorflow-inference:2.10.0-gpu-py39-cu112-ubuntu20.04-sagemaker')
```

Comment:
nit: This method of specifying the image to be used for testing has room for improvement, but if we're unlikely to run this integration test in opt-in regions or in regions like "cn-north-1", I think this is okay for now.

I would however recommend changing the image URI to:
```
763104351884.dkr.ecr.{estimator.sagemaker_session.boto_region_name}.amazonaws.com/tensorflow-inference:2.10-gpu
```

Also, as per the xfail reason below, TF 2.11 saved models are not backward compatible. Will this test work if using a TF 2.10 inference image?

Reply:
Perhaps you could even use

https://github.com/aws/sagemaker-python-sdk/blob/554952eac259979dc714a1a9002653ced342b876/src/sagemaker/image_uris.py#LL41C5-L41C13